build/task-test.js

```diff
@@ -1,17 +1,54 @@
 // Any copyright is dedicated to the Public Domain.
 // http://creativecommons.org/publicdomain/zero/1.0/
+/* eslint no-console: 0 */
 
-import { spawn } from './utils';
+import { spawn, getElectronPath } from './utils';
 import path from 'path';
 
-/* eslint no-console: 0 */
+const MOCHA_TEST_DIRS = ['unit', 'lint', 'webdriver'];
+const DEFAULT_MOCHA_TESTS = path.join(__dirname, '..', 'test',
+                            `+(${MOCHA_TEST_DIRS.join('|')})`, '**', '*.js');
+const DEFAULT_RENDERER_TESTS = path.join(__dirname, '..', 'test', 'renderer', '**', '*.js');
```

Comment:
Maybe split these up into multiple variable assignments so it's easier to read? My brain hurts parsing this.